requirements.txt

```diff
@@ -1 +1 @@
-pypika==0.14.1
\ No newline at end of file
+pypika>=0.14.1
```

Comment:
I think we should do it like `pypika>=0.14.1,<1.0` because if major release will have backward incompatibility it could ruin everything

Reply:
:+1: